mmdet/ops/gcb/context_block.py

```diff
@@ -0,0 +1,100 @@
+import torch
+from torch import nn
+from mmcv.cnn import constant_init, kaiming_init
+
+
+def last_zero_init(m):
+    if isinstance(m, nn.Sequential):
+        constant_init(m[-1], val=0)
+    else:
+        constant_init(m, val=0)
+
+
+class ContextBlock(nn.Module):
+
+    def __init__(self, inplanes, planes, pool, fusions):
```

Comment:
May be better to add a suffix like `method` or `type`?
`pooling_type/pooling_method`
`fusion_types/fusion_methods`